### PR TITLE
cargo-license: init at 0.2.0

### PR DESCRIPTION
**Pull request by basvandijk**
###### Things done

- [ ] Tested using sandboxing ([nix.useSandbox](http://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](http://nixos.org/nix/manual/#sec-conf-file) on non-NixOS)
- Built on platform(s)
   - [ ] NixOS
   - [x] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nix-review --run "nix-review wip"`
- [x] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Assured whether relevant documentation is up to date
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

---

### CODE REVIEW

**Comment by basvandijk** [comment]
@GrahamcOfBorg build cargo-license

File: pkgs/tools/package-management/cargo-license/default.nix [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+    repo = "cargo-license";
+    rev = "v${version}";
+    sha256 = "0xxgl9d695ncrxz29125wag285dwxpwc3fym0ixgj5fqbnkbx75g";
+  };
+
+  cargoPatches = [ ./add-Cargo.lock.patch ];
+
+  cargoSha256 = "0jc84v8fxzzyfkcnfr9vrdblw5vdk54nzpch5lcarzfsarncqxw7";
+
+  meta = with lib; {
```
Comment (JohnAZoidberg) [inline pkgs/tools/package-management/cargo-license/default.nix]: `homepage` please :)
https://github.com/onur/cargo-license

Comment (basvandijk) [reply pkgs/tools/package-management/cargo-license/default.nix]: Oh yes, I'll add one. Thanks for pointing this out.

Comment (basvandijk) [reply pkgs/tools/package-management/cargo-license/default.nix]: Fixed in https://github.com/NixOS/nixpkgs/pull/63772

**Pull request merged**